application/data/tables.py

```diff
@@ -71,11 +76,13 @@ def flat_row_grouped(item, group):
 
 class TableObjectTableBuilder:
     @staticmethod
-    def build(table_object):
+    def build(dimension_table: Table):
+        table_object = dimension_table.table_object
+
         if table_object["type"] == "simple":
```

Comment:
Man this code is such a mess... surely this logic should happen inside `TableObjectDataBuilder.build` when it decides for itself what to put in `table['data']`.  But like you say... "the coming months".  Shoukd we start adding TODOs in the code yet?

Reply:
You'd think so, probably! Maybe I should have put some TODOs. Although I could probably get lost in doing that for days...